File: Golang/Easy/1304_FInd_N_Unique_Integers_Sum_up_to_Zero/solution.go
package findnuniqueintegerssumuptozero

func sumZero(n int) []int {
	output := []int{}
	count := 0
	for i := 1; i <= n; i++ {
		if i == n {
			output = append(output, count*-1)
		} else {
			count += i
			output = append(output, i)
		}
	}
	return output
}

File: Python/Easy/1304_Find_N_Unique_Integers_Sum_up_to_Zero/solution.py
class Solution:
    def sumZero(self, n: int) -> list[int]:
        output = []
        count = 0
        for i in range(1, n+1):
            if i == n:
                output.append(count*-1)
            else:
                output.append(i)
                count += i
        return output